testing/kuttl/e2e/major-upgrade-missing-image/01--valid-upgrade.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Feel like this test could use a README.md to describe the steps. I'm actually a little confused -- do we ever hit the "missing image" error?

(Alternately: what happens if we have a 13 cluster and try to upgrade but have no 14 related image?

Reply:
Yes, because the initial steps use a non-available Postgres version. For the second case, assuming an appropriate upgrade image is used, the cluster would be unable to start until the required image was defined, but upgrade shouldn't be impacted. I've added a README as well.